.env.example

```diff
@@ -1,18 +1,31 @@
-# local, rinkeby, goerli, etc
+# rinkeby, trufflepig
+LOADER=trufflepig
+
+# local, goerli, mainnet
 NETWORK=local
-CHAIN_ID=1337
 
-# trufflepig, network
-LOADER=network
+# chain id
+# local - 1337
+# goerli - 5
+# mainnet - 1
+CHAIN_ID=1337
 
-# .eth for main/devnet, .test for testnet
-COLONY_NETWORK_ENS_NAME=joincolony.eth
+# pinning service pubsub room
+PINNING_ROOM=PINION_DEV_ROOM
 
-SKIP_GAS_STATION_CONFIRM=true
+# verbose logger
 VERBOSE=true
 
-# API keys for development
+# ethplorer key (use `freekey` for rate limited dev trial)
 ETHPLORER_API_KEY=freekey
 
-# Infura ID
-INFURA_ID=blahblah
+# Uncomment to be rate limited with INFURA
```

Comment:
Haha this comment! Well either way they don't enforce the rate limit. But just to clarify, if you don't set this in `.env` it uses our default key from colonyJS

Reply:
We have a default key in colonyJS?

Reply:
Sure do! https://github.com/JoinColony/colonyJS/blob/develop/packages/colony-js-client/src/lib/InfuraProvider/index.js#L8

It's our "public" key, but if Infura start enforcing the daily request limits, then we're probably going to want a private one for the dApp.